CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+
+Before submitting an enhancement suggestion, check if there is already a filed one which provides that enhancement.
+
+First of all, you can validate it with the community, our official channels are:
+
+- [Telegram group](https://t.me/vesperfinance/)
+- [Discord](https://discord.com/invite/Pm8UDaazsK)
+
+How Do I Submit A (Good) Enhancement Suggestion?
+
+General tips:
+
+- Use a clear and descriptive title to identify the suggestion.
+- Provide a step-by-step description of the suggested enhancement in as many details as possible.
+- Provide specific examples to demonstrate the steps. Include copy/pasteable snippets which you use in those examples, as Markdown code blocks.
+- Describe the current behaviour and explain which behaviour you expected to see instead and why.
+- Include screenshots and/or animated GIFs which help you demonstrate the steps or point out the part of Pure Finance which the suggestion is related to. You can use this tool to record GIFs on macOS and Windows, and [this tool](https://github.com/GNOME/byzanz) on Linux.
+- Explain why this enhancement would be useful to most Pure Finance users.
+- Specify the name and version of the OS/Browser/other applicable software you're using.
+
+Then, you can file a new feature by using a template:
+[Feature Template](#)
+
+- Github Feature Request Template
+  [Feature Request Template](#)
+
+### 5. Pull Requests
+
+The process described here has several goals:
+
+- Maintain repository quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible product
+- Enable a sustainable system for Pure Finance's maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+- Follow all instructions in the [Pull Request Template]()
+- Follow the styleguides as stated in step [#2](#2-code-styleguides)
+- If the pull request features a UI improvement, please include a screenshot in order to let maintainers to have a glimpse of it.
+- After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing.
+
+`If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.`
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+
+### 6. Translating guide
+
+We are happy to have people volunteer to translate the App. Just follow the steps as described [here](https://github.com/bloqpriv/vesper-monorepo/wiki).
```

Comment:
This repo is private. Should describe translation guidelines here or in a local wiki page.

Reply:
Local wiki created.